src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/ChatWithCustomData/.template.config/template.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "http://json.schemastore.org/template",
   "author": "Microsoft",
   "classifications": [ "Common", "AI", "Web" ],
-  "identity": "Microsoft.Extensions.AI.Templates.Chat.CSharp",
+  "identity": "Microsoft.Extensions.AI.Templates.WebChat.CSharp",
```

Comment:
Should "AI Chat Web App" be "AI Web Chat App"? I can see it going both ways.

Reply:
Is it "AI Chat" webapp or "AI web chat" app? As in, if there could be an "AI Chat" winapp then it should be the former. I wouldn't think an "AI win chat" app would be very technologically different from an "AI web chat" app....

Reply:
I wanted to focus on scenario first ("AI Chat") in the name and then app type, e.g. ("Web App", or "Web API")